gradle.properties

```diff
@@ -1,2 +1,2 @@
-groovyVersion=2.5.7
+groovyVersion=2.5.4
```

Comment:
downgraded - should we add a classifier to this artifact, to separate it from the original `2.5.4`?

Reply:
That's what the other version number is for here.

Reply:
I don't think I can change that, since it's hard coded on the call site.
It will still try to use the old Groovy version, before it can call the new one (to fetch this jar, I think)